src/commons/build-partial-update-state-command.ts

```diff
@@ -0,0 +1,59 @@
+import { StateDetails } from '../data-types/interfaces';
+import { UpdateItemCommandInput } from '@aws-sdk/client-dynamodb';
+import { getCurrentTimestamp } from './get-current-timestamp';
+import { EventsEnum, MetricNames } from '../data-types/constants';
+import { logAndPublishMetric } from './metrics';
+
+/**
+ * Method to build a Partial of UpdateItemCommandInput
+ * @param newState - new account state object
+ * @param eventName - the name of the event received
+ * @param interventionName - optional intervention name if the event was a fraud intervention
+ */
+export const buildPartialUpdateAccountStateCommand = (
+  newState: StateDetails,
+  eventName: EventsEnum,
+  interventionName?: string,
+): Partial<UpdateItemCommandInput> => {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const base: Record<string, any> = {
```

Comment:
Could we possibly rename this variable? `base` is quite vague and would help with readability.